.eslintignore

```diff
@@ -1,3 +1,4 @@
-**/node_modules
+node_modules/*
+samples/node_modules/*
```

Comment:
You shouldn't need to make that change - what's up here?

Reply:
Have you installed the latest? 😛

Reply:
😅